### PR TITLE
Updated java version for c7 base

### PR DESCRIPTION
**Pull request by bindushree16**
https://github.com/Shippable/pm/issues/11426

image

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -77,7 +77,7 @@ sudo yum install jq-1.5*
 echo "================= Installing Node 8.x ==================="
 . /c7/node/install.sh
 
-echo "================= Installing Java 10.x ==================="
+echo "================= Installing Java 11 ==================="
```
Comment (rageshkrishna) [inline install.sh]: Please remove the version numbers from here. The actual version number is coming from the install.sh file, so you don't really know here what it's going to install.

File: java/install.sh [hunk 2]
```diff
@@ -1,45 +1,33 @@
 #!/bin/bash -e
 
-echo "================= Installing openjdk-10-jdk ==================="
+echo "================= Installing openjdk-11-jdk ==================="
 mkdir -p /usr/lib/jvm && cd /usr/lib/jvm
-wget "https://download.java.net/java/GA/jdk10/10.0.2/19aef61b38124481863b1413dce1855f/13/openjdk-10.0.2_linux-x64_bin.tar.gz"
-tar -xzf openjdk-10.0.2_linux-x64_bin.tar.gz
-mv jdk-10.0.2/ java-10-openjdk-amd64
+wget "https://download.java.net/java/ga/jdk11/openjdk-11_linux-x64_bin.tar.gz"
```
Comment (rageshkrishna) [inline java/install.sh]: The version number should come from an env var.

File: install.sh (hunk 1 above)
Comment (bindushree16) [reply install.sh]: ok

File: java/install.sh (hunk 2 above)
Comment (bindushree16) [reply java/install.sh]: ok

File: java/install.sh [hunk 3]
```diff
@@ -1,45 +1,35 @@
...
+mv jdk-"$OPENJDK_VERSION"/ java-11-openjdk-amd64
 
-sudo update-alternatives --install /usr/bin/java java /usr/lib/jvm/java-10-openjdk-amd64/bin/java 1
-sudo update-alternatives --install /usr/bin/javac javac /usr/lib/jvm/java-10-openjdk-amd64/bin/javac 1
-sudo update-alternatives --set java /usr/lib/jvm/java-10-openjdk-amd64/bin/java
-sudo update-alternatives --set javac /usr/lib/jvm/java-10-openjdk-amd64/bin/javac
+sudo update-alternatives --install /usr/bin/java java /usr/lib/jvm/java-11-openjdk-amd64/bin/java 1
+sudo update-alternatives --install /usr/bin/javac javac /usr/lib/jvm/java-11-openjdk-amd64/bin/javac 1
+sudo update-alternatives --set java /usr/lib/jvm/java-11-openjdk-amd64/bin/java
+sudo update-alternatives --set javac /usr/lib/jvm/java-11-openjdk-amd64/bin/javac
```
Comment (rageshkrishna) [inline java/install.sh]: Shouldn't these also come from the env var?

**Pull request merged**